Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackIntegrationDriver.swift

```diff
@@ -39,59 +35,131 @@ class SentryUserFeedbackIntegrationDriver: NSObject, SentryUserFeedbackWidgetDel
         if let darkThemeOverrideBuilder = configuration.configureDarkTheme {
             darkThemeOverrideBuilder(configuration.darkTheme)
         }
-        
-        if configuration.widgetConfig.autoInject {
-            createWidget()
+
+        if let customButton = configuration.customButton {
+            customButton.addTarget(self, action: #selector(showForm(sender:)), for: .touchUpInside)
```

Comment:
`h`: We should wrap `self` in a weak box, because of this comment in the documentation of `addTarget`:

> The control does not retain the object in the target parameter. It is your responsibility to maintain a strong reference to the target object while it is attached to a control.

If the SentrySDK is closed and uninstalled, I would expect it to dealloc the user feedback integration driver, therefore this can cause `self` to be deallocated, leaving a dangling target action.

Alternative could be unregistering from the `customButton` on `dealloc`.